wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationValues.swift

```diff
@@ -36,7 +36,15 @@ final class ConversationCreationValues {
     var allowGuests: Bool
     var allowServices: Bool
     var enableReceipts: Bool
-    var encryptionProtocol: Feature.MLS.Config.MessageProtocol
+    var encryptionProtocol: Feature.MLS.Config.MessageProtocol {
+        didSet {
+            allowServices = shouldIncludeServices
+        }
+    }
+
+    var shouldIncludeServices: Bool {
+        return !encryptionProtocol.isOne(of: .mls, .mixed)
```

Comment:
Can this use  `encryptionProtocol.supportsBots`?

Reply:
If we were using `Feature.MLS.Config.MessageProtocol`  just to decode the payload, it could be a separate type. But we are using `Feature.MLS.Config.MessageProtocol` in the conversation creation flow, which means we have 2 types with the same values. This looks like code duplication.
I changed the conversation creation flow and use `MessageProtocol` there. As a result, we can re-use `supportsBots` property.